pkg/orchestration/wiring/datadog/plugin.go

```diff
@@ -128,51 +134,16 @@ func constructServiceInstance(resource *orch_v1.StateResource, context *wiringpl
 	return alarmInstanceResource, nil
 }
 
-func generateQuerySpec(context *wiringplugin.WiringContext, alarmType AlarmType, threshold AlarmThresholds, deploymentResourceName smith_v1.ResourceName) QueryParams {
```

Comment:
I have inlined a bunch of things. I think it ready much better when you don't have to jump from func to func to see the whole parameters block.